lms/templates/courseware/course_about.html

```diff
@@ -233,7 +232,7 @@ <h1>
             <li class="important-dates-item">
               <span class="icon fa fa-calendar" aria-hidden="true"></span>
               <p class="important-dates-item-title">${_("Classes Start")}</p>
-              % if isinstance(course_start_date, string_types):
+              % if isinstance(course_start_date, basestring):
```

Comment:
Are you sure this is what we want?
`six` appears to be used here for py2/3 compatibility [1].

> ### six.string_types
> Possible types for text data. This is basestring() in Python 2 and str in Python 3.

[1] https://pythonhosted.org/six/#six.string_types